### PR TITLE
Add GET endpoint to list API tokens

### PR DESCRIPTION
**Pull request by Elhebert**
### What does it do?

It adds a GET endpoints that returns all the API tokens sorted alphabetically by `ApiToken.name`.

### Why is it needed?

This will allow us to retrieve all the tokens in the admin UI.

### How to test it?

The logic is supported by both unit and e2e tests.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

File: packages/core/admin/server/services/api-token.js [hunk 1]
```diff
@@ -2,14 +2,18 @@
...
 
+/**
+ * @typedef {'read-only'|'full-access'} TokenType
+ */
+
 /**
  * @typedef ApiToken
  *
- * @property {number} id
+ * @property {number|string} id
```
Comment (petersg83) [inline packages/core/admin/server/services/api-token.js:12]: Why this change?

File: packages/core/admin/server/tests/admin-api-token.test.e2e.js [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 const { createStrapiInstance } = require('../../../../../test/helpers/strapi');
 const { createAuthRequest } = require('../../../../../test/helpers/request');
+const constants = require('../services/constants');
```
[the remark this answers is not in the document]
Comment (Elhebert) [reply packages/core/admin/server/tests/admin-api-token.test.e2e.js:5]: Yes, that's a good point.

File: packages/core/admin/server/services/api-token.js (hunk 1 above)
Comment (Elhebert) [reply packages/core/admin/server/services/api-token.js:12]: The `id` isn't necessary a number right? IIRC @alexandrebodin pointed that out in my previous PR. When I checked the validation rules for the `id`'s I saw that you have a custom rules that check if it's either a string or a number. So I adapted the jsdoc to reflect that.

Comment (petersg83) [reply packages/core/admin/server/services/api-token.js:12]: I just talked with Alex. So, currently in V4 we only have numbers (in V3 we had string to because we had mongoDB, but not anymore). However, we may allow using uuid in the future so it's not a problem to already say it can be a string. It automatically cast anyway if we put `'1'`.
So we can let `number|string` 👍

**Review by petersg83 — APPROVED** [review]
LGTM :)
